src/index.html

```diff
@@ -19,7 +19,7 @@
     <app-root>
         <div class="mt-5 d-flex justify-content-center">
             <div>
-                <img src="./images/logo-dark@2x.png" class="mb-4 logo" alt="Bitwarden">
+                <div id="loginLogo" class="mb-4 logo" alt="Bitwarden"></div>
```

Comment:
Is it right that this has changed from an `img` to a `div` even though it's still just to display an image?

Reply:
Yeah, I've changed it to be a background image which can be swapped out with the theme in the stylesheet